templates/splash/layout-splashpage.html

```diff
@@ -6,26 +6,26 @@
 		<title>{{ page.title }}</title>
 		<link rel="schema.dcterms" href="http://purl.org/dc/terms/" />
 		<meta content="width=device-width, initial-scale=1" name="viewport" />
-		<meta name="description" content="{{ page.description-en }}" />
-		<meta name="dcterms.description" content="{{ page.description-en }}" />
-		<meta name="dcterms.creator" content="{{ page.creator-en }}" />
+		<meta name="description" content="{{ page.description[ page.lang ] }}" />
+		<meta name="dcterms.description" content="{{ page.description[ page.lang ] }}" />
+		<meta name="dcterms.creator" content="{{ page.creator[ page.lang ] }}" />
```

Comment:
Use the variable `i18nText-lang` instead of `page.lang` because it support multiple format to define the current language of a given page.

Reply:
You would probably need to explain a bit more why we need `i18nText-lang` when `page.lang` is the more usual Jekyll way to do it and kind of leverage the API.